backend/app/core/validation/cclw/law_policy/process_csv.py

```diff
@@ -214,6 +214,10 @@ def validated_input(csv_file: TextIOWrapper) -> DictReader:
 _ValidMetadata = Mapping[str, Mapping[str, Collection[str]]]
 
 
+def import_id_from_csv_row(row: Mapping[str, str]) -> str:
+    return f"CCLW.{row[CATEGORY_FIELD].strip()}.{row[ACTION_ID_FIELD].strip()}.{row[DOCUMENT_ID_FIELD].strip()}"
```

Comment:
Is the category bit here safe? there's a bit more going on when calculating `presented_category`

Reply:
which now I look no longer seems to be used...

Reply:
Is this an issue?

Reply:
it's probably fine